RA_Integration/RA_Dlg_AchEditor.cpp

```diff
@@ -792,6 +799,9 @@ INT_PTR Dlg_AchievementEditor::AchievementEditorProc( HWND hDlg, UINT uMsg, WPAR
 
 			m_BadgeNames.InstallAchEditorCombo( GetDlgItem( m_hAchievementEditorDlg, IDC_RA_BADGENAME ) );
 			m_BadgeNames.FetchNewBadgeNamesThreaded();
+
+			if ( m_pSelectedAchievement != NULL )
+				RepopulateGroupList(m_pSelectedAchievement);
```

Comment:
This ensures the Group list is populated when the editor is opened via double-clicking from the achievements list. It's normally done when the selected achievement changes, but when double-clicking that happens before the list is created, so it can't be populated.